File: src/main/java/br/com/jiratorio/client/config/HolidayClientConfig.java
package br.com.jiratorio.client.config;

import feign.RequestInterceptor;
import org.springframework.context.annotation.Bean;

public class HolidayClientConfig {

    @Bean
    public RequestInterceptor requestInterceptor() {
        return template -> {
            template.query("json", "true");
        };
    }

}

File: jira-library/src/main/kotlin/br/com/jiratorio/jira/client/config/HolidayClientConfig.kt
package br.com.jiratorio.jira.client.config

import feign.RequestInterceptor
import org.springframework.context.annotation.Bean

class HolidayClientConfig {

    @Bean
    fun requestInterceptor() = RequestInterceptor {
        it.query("json", "true")
    }

}
